gait_mapper/utils.py

```diff
@@ -0,0 +1,9 @@
+import tensorflow as tf
+
+
+def sample_latent_features(distribution):
```

Comment:
The sampling function is used only by VAE class, it's also fine to put it to the class as an private method

Reply:
Corrected!